modules/mongodb/mongodb.go

```diff
@@ -0,0 +1,114 @@
+package mongo
+
+import (
+	"context"
+	"time"
+
+	"github.com/netdata/go.d.plugin/agent/module"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+func init() {
+	module.Register("mongodb", module.Creator{
+		Defaults: module.Defaults{
+			UpdateEvery:        module.UpdateEvery,
+			AutoDetectionRetry: module.AutoDetectionRetry,
+			Priority:           module.Priority,
+			Disabled:           false,
```

Comment:
These are defaults. Let's do not set them explicitly.

Reply:
done